JSDemos/Demos/Tabs/Overview/React/App.tsx

```diff
@@ -32,9 +34,20 @@ const App = () => {
   const [showNavigation, setShowNavigation] = React.useState(false);
   const [scrollContent, setScrollContent] = React.useState(false);
   const [fullWidth, setFullWidth] = React.useState(false);
+  const [width, setWidth] = React.useState('auto');
   const [rtlEnabled, setRtlEnabled] = React.useState(false);
   const [widgetWrapperClasses, setWidgetWrapperClasses] = React.useState('widget-wrapper widget-wrapper-horizontal');
 
+  const enforceWidthConstraint = React.useCallback(
```

Comment:
wouldn't it be worth adding a test for the scenario that this code fixes?

Reply:
Already exist: [[link](https://github.com/DevExpress/devextreme-demos/blob/23_2/testing/widgets/tabs/Overview.test.js)]